db/transforms/base.py

```diff
@@ -390,6 +390,48 @@ def _add_aliases_to_summarization_expr_field(
     return summarization
 
 
+class RemoveColumns(Transform):
+    """
+    We're implementing this transform in terms of selecting columns (since Postgres doesn't really
+    support "selecting everything except").
+    """
+    type = "remove"
```

Comment:
`remove` seems like a very generic name, can we call it `remove_columns` or something like that?

I'm not familiar with this area of the codebase so please ignore my comment if they are not helpful or conflict with the style.

Reply:
I considered that. As noted in the PR description, a close sibling of this transform is the `select` transform, which is just as generic, but happens to match an SQL term. That's the rationale behind calling this `remove`.

Standard SQL doesn't do "removing columns", though some variants have `SELECT * EXCEPT ...`, so another idea could be calling this transform `except`. We can also say `remove_columns`, but then let's rename `select` into `select_columns`.

Reply:
`hide` might be a good alternative (and also matches the UI). I think of `remove` as permanent somehow.